charts/ccd-case-management-web/values.preview.template.yaml

```diff
@@ -0,0 +1,22 @@
+tags:
```

Comment:
values.yaml should not include any templated values, the templated values that are currently there should be moved here and sensible default should be set there such as the latest image.

Reply:
👍

Reply:
@smathangi we could address Tim's comment as part of a separate PR dedicated to that issue perhaps? To minimise the risks?

Reply:
Thanks @timja . Can you take a look again please.